docker/launch_example.py

```diff
@@ -11,21 +17,21 @@
 from collections import OrderedDict
 from tvb.datatypes.connectivity import Connectivity
 from tvb.simulator.monitors import Raw
-from tvb_timeseries.model.timeseries import Timeseries
-from tvb_nest.base.config import *
+from tvb_scripts.timeseries.model import Timeseries
+from tvb_nest.config import *
 from tvb_nest.plot.plotter import Plotter
-from tvb_nest.base.simulator import Simulator
-from tvb_nest.tvb_models.reduced_wong_wang_exc_io_inh_i import ReducedWongWangExcIOInhI
-from tvb_nest.nest_models_builders.red_ww_exc_io_inh_i import RedWWExcIOInhIBuilder
-from tvb_nest.interface_builders.red_ww_exc_io_inh_i import RedWWexcIOinhIBuilder
+from tvb_nest.simulator_tvb.simulator import Simulator
+from tvb_nest.simulator_tvb.model_reduced_wong_wang_exc_io_inh_i import ReducedWongWangExcIOInhI
+from tvb_nest.simulator_nest.models_builders.red_ww_exc_io_inh_i import RedWWExcIOInhIBuilder
+from tvb_nest.interfaces.builders.red_ww_exc_io_inh_i import RedWWexcIOinhIBuilder
 
 config = Config(output_base="outputs/")
 config.figures.SAVE_FLAG = False
 config.figures.SHOW_FLAG = False
 config.figures.MATPLOTLIB_BACKEND = "Agg"
 plotter = Plotter(config)
 
-connectivity = Connectivity.from_file(os.path.join(DEFAULT_SUBJECT_PATH, DEFAULT_CONNECTIVITY_ZIP))
+connectivity = Connectivity.from_file(os.path.join(Config.DEFAULT_SUBJECT_PATH, Config.DEFAULT_CONNECTIVITY_ZIP))
 connectivity.configure()
 #plotter.plot_tvb_connectivity(connectivity)
```

Comment:
Pls do not update launch_example. yet. It is in direct relation with HBP Collab usages, and the release Docker image on Docker Hub. We will need to upgrade all of those in correlation with this file. Currently it is fine to remain outdated, this file

Reply:
Ok. This file will be reverted.